addons/point_of_sale/models/pos_session.py

```diff
@@ -1957,20 +1959,35 @@ def _loader_params_pos_category(self):
         domain = []
         if self.config_id.limit_categories and self.config_id.iface_available_categ_ids:
             domain = [('id', 'in', self.config_id.iface_available_categ_ids.ids)]
-
         return {'search_params': {'domain': domain, 'fields': ['id', 'name', 'parent_id', 'child_id', 'write_date', 'has_image']}}
 
     def _get_pos_ui_pos_category(self, params):
         return self.env['pos.category'].search_read(**params['search_params'])
 
+    def _loader_params_pos_combo(self):
+        products = self._context.get('loaded_data')['product.product']
+        combo_ids = sum([product.get('combo_ids') for product in products], [])
```

Comment:
@vlst-odoo can `combo_ids` have duplicates?

Reply:
I consulted one of the orm team members and he confirmed that browse is better because it can hit the cache.
Also, it's better to dedup the ids so that the result of `read` has no duplicates.

Reply:
ok, makes sense